src/briefcase/platforms/android/gradle.py

```diff
@@ -358,6 +359,22 @@ def add_options(self, parser):
             help="Shutdown the emulator on exit",
             required=False,
         )
+        parser.add_argument(
+            "--forward-port",
+            action="append",
+            dest="forward_ports",
+            type=int,
+            help="Forward the specified port from host to device.",
+            required=False,
```

Comment:
I don't know why the existing code specifies required=False so much when that's the default, but there's no need to perpetuate that.

```suggestion
```